src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/BaseHandlerTestSetup.java

```diff
@@ -95,4 +99,18 @@ protected FinremCallbackRequest buildFinremCallbackRequest(String testJson) {
             throw new RuntimeException(e);
         }
     }
+
+    protected void assertCanHandle(CallbackHandler handler, CallbackType expectedCallbackType, CaseType expectedCaseType,
```

Comment:
This is a great idea. However, it doesn't feel like this is the right class for it. How about creating a new class e.g. Assertions. Going forward we could add to it as suggested here https://www.baeldung.com/assertj-custom-assertion

Reply:
Updated.